src/app/core/data/request.service.ts

```diff
@@ -54,6 +55,25 @@ export class RequestService {
     return pathSelector<CoreState, string>(coreSelector, 'index', IndexName.UUID_MAPPING, uuid);
   }
 
+  private uuidsFromHrefSubstringSelector(selector: MemoizedSelector<any, IndexState>, name: string, href: string): MemoizedSelector<any, string[]> {
+    return createSelector(selector, (state: IndexState) => this.getUuidsFromHrefSubstring(state, name, href));
+  }
+
+  private getUuidsFromHrefSubstring(state: IndexState, name: string, href: string): string[] {
```

Comment:
Could we add typedocs or inline comments to this private method. It's not entirely clear to me how this is parsing out the UUIDs.

Reply:
Added

Reply:
@artlowel : It looks like these typedocs/comments were not yet added.  I see new comments on a different private method here, but not on the `getUuidsFromHrefSubstring()` method.